### ISSUE TITLE
ItemGroup's title should has the same paddingLeft logic with Item

### ISSUE THREAD

**Issue by micate** [react-component/menu#198]
Item has [a `paddingLeft` logic based on `level`](https://github.com/react-component/menu/blob/master/src/MenuItem.jsx#L183):
```js
if (props.mode === 'inline') {
  style.paddingLeft = props.inlineIndent * props.level;
}
```
I think ItemGroup's title also need this logic, which unable to use css/less to implement.

**Comment by picodoth** [comment]
@micate do you mind to open a PR for it? :)

**Comment by micate** [comment]
> @micate do you mind to open a PR for it? :)

added.

**Comment by micate** [comment]
@picodoth added.

**Comment by picodoth** [comment]
available in 7.4.10
